test/ufuzz.js

```diff
@@ -582,6 +582,14 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
       case 30:
         return '(' + createObjectLiteral(recurmax, COMMA_OK, stmtDepth, canThrow) + '.' +
             SAFE_KEYS[rng(SAFE_KEYS.length)] + ") ";
+      case 31:
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
+        return '(' + name + ' && ' + name + '[' +
+            createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
+      case 32:
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
```

Comment:
> you still don't need typeof here because k && k.foo = 123 would not throw under all circumstances 

But it throws in some circumstances. We should limit throwing only to when `canThrow` is true.